src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -513,4 +513,23 @@ Use the dropdown to view and switch to other projects this file may belong to.</
   <data name="Unknown" xml:space="preserve">
     <value>&lt;Unknown&gt;</value>
   </data>
+  <data name="IntellisenseBuildFailedMessage" xml:space="preserve">
+    <value>Encountered some issues while loading the project. this will prevent us from providing some features such as full solution analysis for this project and any projects that depend on this project.</value>
+  </data>
+  <data name="IntellisenseBuildFailedTitle" xml:space="preserve">
+    <value>Project loading failed.</value>
+  </data>
+  <data name="IntellisenseBuildFailedDescription" xml:space="preserve">
+    <value>To see what caused the issue, please try below.
```

Comment:
clicking "code" will do bing search for the error, it will find those blogs if there is any. and that way we don't need to worry about blog coming and going.

well, it might be awful, but it being right there actually help as well. one can do bing search to get better explanation as well. it doesnt block that.